src/main/java/UserSearchUI.java

```diff
@@ -0,0 +1,60 @@
+/**
+ * Provides the UI elements
+ */
+import javax.swing.*;
+import java.awt.GridLayout;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+
+
+public class UserSearchUI implements UserPresenter{
+    private JLabel label;
+    UserDatabase db = UserDatabase(accounts);
```

Comment:
Here, the view directly accesses the persistence mechanism (userDatabase). This is likely okay, although it differs from the example clean architecture diagram from lecture (attached below) where the use case interactor queries the persistence layer.

![image](https://user-images.githubusercontent.com/18428059/201395133-bc23d4e5-dfdb-4fe1-9a00-0feb9134bb10.png)

Reply:
Yes - I am actually a little confused about where else to put that - I think I'm gonna ask Myles on Monday